migration/migration.go

```diff
@@ -218,11 +218,10 @@ func getMigrations() migrations {
 		workitem.SystemExperience.String(),
 		workitem.SystemFeature.String(),
 		workitem.SystemScenario.String(),
-		workitem.SystemBug.String(),
-	)})
+		workitem.SystemBug.String())})
 
-	// Version 35
-	m = append(m, steps{executeSQLFile("035-add-icon-to-wit.sql")})
+	// Version 36
+	m = append(m, steps{executeSQLFile("036-add-icon-to-wit.sql")})
```

Comment:
It is fully dynamic already and the versioning is just for you as a human to inspect the folder of migrations and see what comes after what. The version is calculated by the position in the `m` array.